jdaviz/configs/specviz2d/plugins/spectral_extraction/spectral_extraction.py

```diff
@@ -354,6 +372,9 @@ def user_api(self):
                                            'export_bg', 'export_bg_img', 'export_bg_sub',
                                            'ext_dataset', 'ext_trace', 'ext_type',
                                            'ext_width', 'ext_add_results',
+                                           'self_prof_n_bins',
+                                           'self_prof_interp_degree_x',
+                                           'self_prof_interp_degree_y',
```

Comment:
should `horne_ext_profile` also be exposed?  (If so, should also be added to the docstring)

Reply:
I'd like that.

Reply:
I'd like that.